### PR TITLE
Update readme and small bug fix

### PR DESCRIPTION
**Pull request by ruowan**

### CODE REVIEW

File: lib/testScenario/bodyTransformer.ts [hunk 1]
```diff
@@ -101,9 +101,9 @@ export class BodyTransformer {
 
     if (typeof dst === "object" && typeof src === "object") {
       const result: any = { ...dst };
```
Comment (jianyexi) [inline lib/testScenario/bodyTransformer.ts:103]: what if dst is null and src is not null ?

Comment (ruowan) [reply lib/testScenario/bodyTransformer.ts:103]: It will not entry this code. So it's safe

Comment (jianyexi) [reply lib/testScenario/bodyTransformer.ts:103]: could you add comment here

File: ChangeLog.md [hunk 2]
```diff
@@ -5,6 +5,8 @@
 - Oav runner support specifying option 'runId' with option 'from' and/or 'to' to debug.
 - Oav runner support using env variable 'TEST_SCENARIO_JSON_ENV' to override variables in env.json.
 - Replace Oav runner option 'cleanUp' with 'skipCleanUp'.
+- Update README.md add gif to show API test
```
Comment (raych1) [inline ChangeLog.md:8]: Why not bump a new version for these new changes?

File: README.md [hunk 3]
```diff
@@ -17,17 +16,39 @@ You can install the latest stable release of node.js from [here](https://nodejs.
...
 ```bash
-npm install -g oav
+npm install -g oav@latest
 ```
 
+![](./documentation/installOav.gif)
+
+### Run API test
+
+OAV support run API test against Azure and validate request and response. You could define test scenario file which compose with serval swagger example file and then use oav to run it. For more details about API test, please refer to this [API testing doc](https://github.com/Azure/azure-rest-api-specs/tree/test-scenario-main/documentation/test-scenario).
```
Comment (raych1) [inline README.md:26]: Typo of `several`.

File: ChangeLog.md (hunk 2 above)
Comment (ruowan) [reply ChangeLog.md:8]: 2.7.0 is not released yet.

File: lib/testScenario/reportGenerator.ts [hunk 4]
```diff
@@ -420,9 +420,8 @@ export class ReportGenerator {
         })
         .filter((it) => it !== undefined) as ResponseDiffItem[];
       return delta;
-    } catch (err) {
-      console.log(err);
```
Comment (raych1) [inline lib/testScenario/reportGenerator.ts:420]: Do you mean log will cause exception?

Comment (ruowan) [reply lib/testScenario/reportGenerator.ts:420]: Yes. Do not output error message here. It make confuse for customer

**Review by raych1 — APPROVED** [review]

**Pull request merged**